src/licensedcode/data/licenses/cc-plus-cal.yml

```diff
@@ -0,0 +1,12 @@
+key: cc-plus-cal
```

Comment:
Can you add the version to this and rename that to this key and file names to `corporate-accountability-commercial-1.1`?